@AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/simpleRotor/fn_simpleRotorTail.sqf

```diff
@@ -41,9 +41,9 @@ private _rtrPowerScalarTable    = [
                                   ,[6000, 1.377]
                                   ,[8000, 1.284]
                                   ];
-private _rtrThrustScalar_min    = -0.0625;
-private _rtrThrustScalar_max    =  0.0581;
-private _rtrThrustScalar_med    = (_rtrThrustScalar_min + _rtrThrustScalar_max) / 2;
+private _rtrThrustScalar_min    = -0.125;
+private _rtrThrustScalar_max    =  0.1162;
+private _rtrThrustScalar_med    = -0.0022;//(_rtrThrustScalar_min + _rtrThrustScalar_max) / 2;
```

Comment:
Interesting. Is there a reason it isn't the mean any more? Just the asymmetry of the rotor?

Old bit can be removed

Reply:
This allows me to have more thrust for left pedal vs. right and balance the tail rotor on the ground. I still need to tweak the value actually. The helicopter will rotate on the groun under certain conditions.